### DISCUSSION TITLE
RspamD support and 2.1

### DISCUSSION THREAD

**Discussion by Skywrath-Mage** [postalserver/postal#1660] [Help with using Postal]
Hello, 

I see that 2.1 is out in the changelog but I can't update it or install it with the CLI. 

Did you guys ware able to do it? and if yes, how? 

(the main reason that i want to switch to 2.1 is the support for rspamd)

**Comment by willpower232** [comment]
You will need to follow the documentation as it is run in a completely different way https://docs.postalserver.io/install/upgrade-to-v2

**Reply by Skywrath-Mage** [reply]
that is from 1.x to 2.0 but there is no way to update to 2.1
I even try a new install but is not possible to install 2.1; the script will just install 2.0

**Reply by phache** [reply]
Me too, I try `postal upgrade` as in the documentation but it doesn't go to 2.1, it stays on 2.0. Wasn't sure if some sort of manual intervention is required before running that command...

**Comment by Skywrath-Mage** [comment]
I have found how to do it, you have to provide the version that you want to install with `postal set-version <version>` like` postal set-version 2.1.0 `


but this is not documented, I have to read the bash manager/configurator script, that is actually shared on the install repository https://github.com/postalserver/install/blob/main/bin/postal#L82

**Reply by willpower232** [reply]
thank you for clarifying, @adamcooke does the installer need to track the current image version manually as a separate process?

**Reply by phache** [reply]
Thanks, that almost worked for me, what I found that I had to do in the end was:

- `postal set-version 2.1.0`
- `postal upgrade 2.1.0`

So basically I had to specify the version in both those commands, and then everything upgraded OK.
